### ISSUE TITLE
LightningTalks: Format and organize video titles on vimeo for consistency 

### ISSUE THREAD

**Issue by jpailthorpe** [engagementgamelab/boston-civic-media#39]
though titles have been shortened to meet length limits #56  , they still are widely uneven in terms of formatting and framing. 

I just added all BCM videos into a vimeo channel with branding and correct link. However, I noticed that the video titles vary widely in terms of format, sometimes mentioning titles, sometimes mentioning people, sometimes framing each with "Case study lightning talk". 

To a complete outside audience, most of these titles don't make much sense. I think we can better title these to include enough context to encourage viewing, and at the same time, make a consistent title format. 

https://vimeo.com/channels/bostoncivicmedia

**Comment by kanarinka** [comment]
Agreed - I suggest assigning the task of revising titles to Sean or Becky

**Comment by jpailthorpe** [comment]
Assigning to sean and sean. If you update these titles + descriptions directly in vimeo, you don't need to do anything with the site as they pull automatically.
